trace/trace.go

```diff
@@ -185,6 +189,22 @@ func EsSpan(ctx context.Context, opName, cmp, user, uri, method, body string, ho
 	return sp
 }
 
+// RedisSpan starts a new Redis child span with specified tags.
```

Comment:
This method is redis specific and should live in the redis package.
ES should also.. we missed this...